<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qCjyxSimpleMarkupsWidget</class>
 <widget class="qCjyxWidget" name="qCjyxSimpleMarkupsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>195</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="qDMMLNodeComboBox" name="MarkupsNodeComboBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="nodeTypes">
        <stringlist>
         <string>vtkDMMLMarkupsFiducialNode</string>
        </stringlist>
       </property>
       <property name="showHidden">
        <bool>true</bool>
       </property>
       <property name="noneEnabled">
        <bool>true</bool>
       </property>
       <property name="addEnabled">
        <bool>true</bool>
       </property>
       <property name="removeEnabled">
        <bool>true</bool>
       </property>
       <property name="editEnabled">
        <bool>false</bool>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="qCjyxMarkupsPlaceWidget" name="MarkupsPlaceWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="MarkupsControlPointsTableWidget"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qDMMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qDMMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qCjyxWidget</class>
   <extends>QWidget</extends>
   <header>qCjyxWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qCjyxMarkupsPlaceWidget</class>
   <extends>qCjyxWidget</extends>
   <header>qCjyxMarkupsPlaceWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Resources/qCjyxMarkupsModule.qrc"/>
  <include location="../../../../../../Libs/DMML/Widgets/Resources/qDMMLWidgets.qrc"/>
  <include location="../../../../../../Base/QTGUI/Resources/qCjyxBaseQTGUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qCjyxSimpleMarkupsWidget</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>MarkupsNodeComboBox</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qCjyxSimpleMarkupsWidget</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>MarkupsPlaceWidget</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
